conda_requirements.yml

```diff
@@ -1,33 +1,130 @@
-# This file may be used to create an environment using:
-# $ conda env create --name <env_name> --file conda_requirements.yml
-name: conee_env
+name: bootleg-dev
 channels:
-  - pytorch
   - conda-forge
   - defaults
 dependencies:
-  - argh=0.26.2
-  - ipdb=0.12.3
-  - jsonlines=1.2.0
-  - marisa-trie=3.1.1
-  - matplotlib=2.2.4
-  - mock==3.0.5
-  - networkx=2.4
-  - nltk=3.4.5
-  - notebook=6.0.3
-  - numpy=1.17.4
-  - pandas=0.25.3
-  - pytest=5.3.0
-  - python=3.7.3
-  - scikit-learn=0.22
-  - scipy=1.3.1
-  - seaborn=0.9.0
-  - tabulate=0.8.7
-  - torch=1.6.0
-  - tagme=0.1.3
-  - tensorboardx=2.0
-  - transformers=3.0.1
-  - torch=1.6.0
-  - torchtext=0.4.0
-  - tqdm=4.51.0
-  - ujson=1.35
+  - python>=3.6
+  - ca-certificates=2020.12.5=h033912b_0
+  - certifi=2020.12.5=py38h50d1736_1
+  - libcxx=10.0.0=1
+  - libedit=3.1.20191231=h1de35cc_1
+  - libffi=3.3=hb1e8313_2
+  - marisa-trie=0.7.5=py38hc84c608_1001
+  - ncurses=6.2=h0a44026_1
+  - openssl=1.1.1i=h35c211d_0
+  - pip=21.0=pyhd8ed1ab_1
+  - python_abi=3.8=1_cp38
+  - readline=8.0=h1de35cc_0
+  - setuptools=49.6.0=py38h50d1736_3
+  - sqlite=3.33.0=hffcf06c_0
+  - tk=8.6.10=hb0a8c7a_0
+  - tzdata=2020f=h52ac0ba_0
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h1de35cc_0
+  - zlib=1.2.11=h1de35cc_3
+  - pip:
+    - absl-py==0.11.0
+    - appnope==0.1.2
+    - argh==0.26.2
+    - async-generator==1.10
+    - attrs==20.3.0
+    - backcall==0.2.0
+    - bleach==3.2.3
+    - blis==0.7.4
+    - cachetools==4.2.1
+    - catalogue==1.0.0
+    - chardet==4.0.0
+    - click==7.1.2
+    - cymem==2.0.5
+    - decorator==4.4.2
+    - defusedxml==0.6.0
+    - entrypoints==0.3
+    - filelock==3.0.12
+    - future==0.18.2
+    - google-auth==1.24.0
+    - google-auth-oauthlib==0.4.2
+    - grpcio==1.35.0
+    - idna==2.10
+    - ipdb==0.12.3
+    - ipykernel==5.4.3
+    - ipython==7.19.0
+    - ipython-genutils==0.2.0
+    - jedi==0.18.0
+    - jinja2==2.11.2
+    - joblib==1.0.0
+    - jsonlines==1.2.0
+    - jsonschema==3.2.0
+    - jupyter-client==6.1.11
+    - jupyter-core==4.7.0
+    - jupyterlab-pygments==0.1.2
+    - markdown==3.3.3
+    - markupsafe==1.1.1
+    - mistune==0.8.4
+    - mock==3.0.5
+    - more-itertools==8.6.0
+    - murmurhash==1.0.5
+    - nbclient==0.5.1
+    - nbconvert==6.0.7
+    - nbformat==5.1.2
+    - nest-asyncio==1.5.1
+    - networkx==2.4
+    - nltk==3.4.5
+    - notebook>=6.1.5
+    - numpy==1.19.5
+    - oauthlib==3.1.0
+    - packaging==20.8
+    - pandas==0.25.3
+    - pandocfilters==1.4.3
+    - parso==0.8.1
+    - pexpect==4.8.0
+    - pickleshare==0.7.5
+    - plac==1.1.3
```

Comment:
Do we need all these packages for conda? Some of them I don't recognize at all, like plac or pluggy?

Reply:
These are just the ones that were created when I tried to install everything in the pip requirements, starting from a new conda environment